cobc/parser.y

```diff
@@ -23,7 +23,7 @@
 
 %defines
 %verbose
-%error-verbose
+%define parse.error verbose
```

Comment:
This is a fix of a Bison warning, but the fix is only possible with "more recent" versions of Bison (a GC4 thing).